docs/configuration.md

```diff
@@ -2013,7 +2013,6 @@ Apart from these, the following properties are also available, and may be useful
     </tr>
 </table>
 
-
```

Comment:
If you have to change the pull request again, I'd revert this, but no need to change it only for this

Reply:
Restored the change.